### DISCUSSION TITLE
Linux - Sharing freeRDP in video chat

### DISCUSSION THREAD

**Discussion by RobLW** [FreeRDP/FreeRDP#11012] [Q&A]
Hello,
A bit of an odd one here. I'm using teams-for-linux, and when I share my screen it generally works fine. However when I switch to the freeRDP app it then only shares a grey screen. 
now if I am in a 1-2-1 video chat with someone and share the screen it seems to work. But not when I have multiple people on the call, and not if I join a pre-arranged meeting (room?) even if it's only with 1 other person.

Initially I assumed this is down to Teams-for-linux but I cannot find a reason for it, nor can they. So hoping there might be some idea's here?
could it be somehow be related to a different codec being used when sharing the screen to multiple users to improve compression maybe?

**Comment by akallabeth** [comment]
no idea what that could be (does work fine with e.g. jitsi) and there is no codec involved,
`xfreerdp` is a normal X11 application with bitmap updates.

it is basic `X11` though, no `gtk`, `qt` or whatever, so maybe that is the bug in teams?
